river/metrics/cluster/sd_validation.py

```diff
@@ -0,0 +1,175 @@
+import math
+
+from river import stats, utils
+
+from . import base
+
+__all__ = ["SD"]
+
+
+class SD(base.InternalClusMetric):
+    """The SD validity index (SD).
+
+    The SD validity index (SD) [^1] is a more recent clustering validation measure. It is composed of
+    two terms:
+
+    * Scat(NC) stands for the scattering within clusters,
+
+    * Dis(NC) stands for the dispersion between clusters.
+
+    Like DB and SB, SD measures the compactness with variance of clustered objects and separation
+    with distnace between cluster centers, but uses them in a different way. The smaller the value
+    of SD, the better.
+
+    In the original formula for SD validation index, the ratio between the maximum and the actual
+    number of clusters is taken into account. However, due to the fact that metrics are update live
+    along with the data stream, this ratio will be automatically set to default as 1.
```

Comment:
Suggestion:

"(...) due to the fact that metrics are updated in an incremental fashion, this ratio (...)"

Reply:
Really nice suggestion! Thank you so much!!
Resolved.